.github/workflows/run.yml

```diff
@@ -45,13 +45,13 @@ jobs:
         run: bin/dfx-sns-demo
       - name: Create and finalize an additional SNS
         run: |
-          ./bin/dfx-sns-demo-mksns
```

Comment:
Can we use the long flag names in our own scripts? IMO the short flags are only for using manually or for things we are so used to that we know the short name better than the long name.

Reply:
Good point, replaced.